html/elements/datalist.json

```diff
@@ -18,19 +18,9 @@
             "edge": {
               "version_added": "12"
             },
-            "firefox": [
-              {
-                "version_added": "110",
-                "partial_implementation": true,
-                "notes": "The `date` and `time` input types are not supported."
-              },
-              {
-                "version_added": "4",
-                "version_removed": "110",
-                "partial_implementation": true,
-                "notes": "The `<datalist>` element will only create a dropdown for textual types, such as `text`, `search`, `url`, `tel`, `email` and `number`. The `date`, `time`, `range` and `color` types are not supported."
```

Comment:
Not all of the types mentioned in the note are represented by the subfeatures. The PR title had me anticipating keys like this:

- html.elements.datalist
- html.elements.datalist.input_type_color
- html.elements.datalist.input_type_date
- html.elements.datalist.input_type_email
- html.elements.datalist.input_type_number
- html.elements.datalist.input_type_range
- html.elements.datalist.input_type_search
- html.elements.datalist.input_type_tel
- html.elements.datalist.input_type_text
- html.elements.datalist.input_type_time
- html.elements.datalist.input_type_url

Reply:
To clarify, this PR only extracts subfeatures for types currently mentioned data, without researching/collecting/adding data for other types not currently mentioned.